src/jobs/deploy-s3.yml

```diff
@@ -1,9 +1,78 @@
-description: >
+description: |
   This job deploys an application to aws s3.
 
 executor: node/default
 
+parameters:
```

Comment:
Chris and I have discussed putting all required parameters at the top of the parameter list so that developers know immediately what parameters are required. Could you also do this? Add comments as well.